Standards/scs-0302-v1-domain-manager-role.md

```diff
@@ -1,17 +1,27 @@
 ---
 title: Domain Manager configuration for Keystone
 type: Standard
-status: Draft
+status: Stable
+stabilized_at: 2024-08-30
```

Comment:
Please do not forget to update the date, when merging :)

Reply:
Oh and did we discuss to stabilize this?

Reply:
Here: https://github.com/SovereignCloudStack/minutes/blob/main/iaas/20240911.md#domain-manager-standard-markus-hentsch